### PR TITLE
Fix tabs on Task Details page

### LINKED ISSUE #551: Tab description disappears on mouseover
`http://allready-d.azurewebsites.net/Admin/Task/Details/66?activityId=14`

When you mouseover either of the tabs, the description disappears.

image

### PR DESCRIPTION
**Pull request by glittle**
Closes #551

### CODE REVIEW

**Comment by tonysurma** [comment]
@mheggeseth @dpaquette can we get a peek at this?  Also is there a way to drop the two merge commits (didn't look at them just saw the titles)

**Comment by mheggeseth** [comment]
I'll look at it.

**Comment by mheggeseth** [comment]
The merge commits will only go away if @glittle does a rebase.

File: AllReadyApp/Web-App/AllReady/wwwroot/css/site.css [hunk 1]
```diff
@@ -115,6 +115,26 @@ ul.nav a:focus {
     text-decoration: underline;
 }
 
+ul.nav.nav-tabs > li > a:hover 
+{
+    /* issue 551 */
+    color: #000;
+}
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/wwwroot/css/site.css]: The `:focus` pseudo-selector also needs to be applied here.
image

**Comment by glittle** [comment]
Please go ahead with this. I don't expect to work on this in the near future...

**Comment by glittle** [comment]
As for the merge commits, we tried to remove them, but couldn't figure out how to do it. Rebasing seemed to make it worse!

**Pull request merged**